### ISSUE TITLE
Include headers in smarthome handler arguments

### ISSUE THREAD

**Issue by mandrewpowers** [actions-on-google/actions-on-google-nodejs#153]
Even though the documentation says Smart Home Apps are not supported by this library, the current version has code to handle it.

Unfortunately the authentication never reaches the handler. Only the request body is passed to the handler so I had no way to perform actions for a specific user without re-creating the `smarthome` object with every request to utilize closures or modify the body before it was passed to the handler.

**Comment by Fleker** [comment]
We just released the new service for smart home integrations, and I didn't finish updating the documentation. It's still very early, and I appreciate the feedback.

To confirm, the feature you want is basically getting the HTTP headers as an object in the smarthome functions, such that:

```
const app = smarthome({});
app.onExecute(body => {
  const headers = body.headers;
  const headerArgument = headers.argument
  return {
    requestId: 'ff36...',
    payload: {
      ...
    }
  }
})
```

**Comment by Fleker** [comment]
This has been resolved in 6a5820edbc52aea730195e43f5c61f5164fdc2bc.
